framework/test/CMakeLists.txt

```diff
@@ -100,6 +102,25 @@ add_executable(${_test_driver} ${_srcs} ${_additional_srcs} ${_third_party_srcs}
 add_dependencies(${_test_driver} ${US_RCC_EXECUTABLE_TARGET})
 set_property(TARGET ${_test_driver} APPEND PROPERTY COMPILE_DEFINITIONS US_BUNDLE_NAME=main)
 set_property(TARGET ${_test_driver} PROPERTY US_BUNDLE_NAME main)
+
+# Disable deprecation warnings in the test code.
+if (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
+    ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang") OR
+    ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU"))
+  #target_compile_optinons(${_test_driver} PUBLIC "-Wno-deprecated-declarations")
+  set_source_files_properties(BundleActivatorTest.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
```

Comment:
Can't we just do this
```
set_source_files_properties(${_srcs} PROPERTIES COMPILE_FLAGS -Wno-error=deprecated-declarations)
```
instead of the repetitive lines and the complete disablement of the warning?